packages/s2-core/__tests__/spreadsheet/hidden-columns-spec.ts

```diff
@@ -554,6 +554,39 @@ describe('SpreadSheet Hidden Columns Tests', () => {
         expect(leafNodes.some((node) => node.id === nodeId)).toBeFalsy();
         expect(leafNodes).toHaveLength(5);
       });
+
+      test('hiding the column totals should not hide the row totals', () => {
```

Comment:
两个测试断言一样用 `test.each` 即可

```suggestion
      test.each(['grid', 'tree'])('hiding the column totals should not hide the row totals for %s mode', (hierarchyType) => {
      
        sheet.setOptions({ hierarchyType);
        sheet.render(false)
```

Reply:
done